sktime/datatypes/_vectorize.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
```

Comment:
Since when have we had a copyright and is this now needed on all files?

Reply:
I thought we always had this? Just inconsistently added.